logging/logging.go

```diff
@@ -298,13 +298,75 @@ func detectGAEResource() *mrpb.MonitoredResource {
 	}
 }
 
+func isCloudRun() bool {
+	_, config := os.LookupEnv("K_CONFIGURATION")
```

Comment:
is this returning false when the variable is null or is empty?

Reply:
Correct, it returns false when the envvar doesn't exist